app/common/vault_api.py

```diff
@@ -13,9 +13,11 @@
 class Vault:
     def __init__(self, logger,
                  owner=None,
-                 repo=None):
+                 repo=None,
+                 cluster_name=''):
         self.owner = owner
         self.repo = repo
+        self.mount_point = f'kubernetes-{cluster_name}'
```

Comment:
So u know both cluster_name and mount point. Cool.

Reply:
sure